shop/src/app/components/presentational/edit-product-view/edit-product-view.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="all-container">
+    <form [formGroup]="productForm" id="product-edit">
+        <div class="form-container">
+            <div class="labels">
+                <label for="name">Product name: </label>
+                <!-- <label for="category">Category: </label> -->
```

Comment:
Can you please remove the commented code if it's no longer used?

Reply:
Done